MapboxDirections/MBAttribute.h

```diff
@@ -3,36 +3,29 @@
 /**
  Attributes are metadata information for a route leg.
  
- When most of the attributes are specified, the resulting route leg contains one attribute value for each segment in leg, where a segment is the straight line between two coordinates in the route leg’s full geometry. When the `MBAttributeOpenStreetMapNodeIdentifier` attribute is specified, the corresponding `RouteLeg` property contains one more value than each of the other attributes.
+ When most of the attributes are specified, the resulting route leg contains one attribute value for each segment in leg, where a segment is the straight line between two coordinates in the route leg’s full geometry.
```

Comment:
Replace “most of the attributes are specified” with “any of the attributes is specified”, now that there’s no longer an exception to the rule.